omod-dstu3/src/test/java/org/openmrs/module/fhir2/web/servlet/BaseFhirProvenanceResourceTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.web.servlet;
+
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Date;
+
+import org.hl7.fhir.convertors.VersionConvertor_30_40;
+import org.hl7.fhir.dstu3.model.DomainResource;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.Practitioner;
+import org.hl7.fhir.r4.model.Provenance;
+import org.hl7.fhir.r4.model.Reference;
+import org.mockito.Mock;
+import org.openmrs.User;
+import org.openmrs.module.fhir2.FhirConstants;
+import org.openmrs.module.fhir2.api.translators.PractitionerReferenceTranslator;
+
+public abstract class BaseFhirProvenanceResourceTest<T extends DomainResource> {
```

Comment:
Instead of making a fork of this class, maybe it makes more sense to call the version here `BaseFhirR3ProvenanceResourceTest` and have it extend `BaseFhirProvenanceResourceTest` and then make the changes that we need here. This might involve changing `BaseFhirProvenanceResourceTest`, but that's ok.